air-quality-backend/system_tests/in_situ_etl_suite/open_aq_etl_tests.py

```diff
@@ -24,6 +28,18 @@
 load_dotenv()
 
 
+@mock.patch.dict(os.environ, {"OPEN_AQ_CITIES": "London"})
+def test__in_situ_etl__calling_actual_api_returns_values_and_stores():
+    query = {"name": "London"}
+    delete_database_data(collection_name, query)
+
+    main()
+
+    results = get_database_data(collection_name, query)
+    assert results[0]["name"] == "London"
+    assert results[-1]["name"] == "London"
```

Comment:
is there a reason you are checking the returned results by checking the first and last element? rather than checking the element and that the results length is 1?

Reply:
In this case we'll have more than 1 result as this isn't mocking the call to open AQ. There will be a lot of results back but we don't really know what they'll be, only that they'll be for London. Happy to add any other checks if you think it worthwhile